### PR TITLE
Material-ui update and dummy header buttons

### PR DESCRIPTION
**Pull request by lukaszczernal**

### CODE REVIEW

File: src/assets/styles/imports.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Fonts
+@import url(https://fonts.googleapis.com/icon?family=Material+Icons);
+@import url('https://fonts.googleapis.com/css?family=Roboto');
```
Comment (szamanm) [inline src/assets/styles/imports.scss]: can we unify it and decide whether we want it with '' or not?

File: src/components/NavigationBar/NavigationBar.tsx [hunk 2]
```diff
@@ -35,6 +39,18 @@ const NavigationBar = decorate<OwnProps>(
             >
               Administration
             </Typography>
+            <Button>
```
Comment (szamanm) [inline src/components/NavigationBar/NavigationBar.tsx]: from specs: https://material-ui-next.com/demos/app-bar/

I think we should add "color="contrast""

**Review by szamanm — COMMENTED** [review]
And maybe add "l" to the commit/review name

File: src/components/NavigationBar/NavigationBar.tsx (hunk 2 above)
Comment (lukaszczernal) [reply src/components/NavigationBar/NavigationBar.tsx]: color="inherit" worked but only for flat buttons. need to investigate more

**Review by szamanm — APPROVED** [review]

**Pull request merged**